pkg/pillar/iptables/ssh.go

```diff
@@ -95,3 +96,36 @@ func dropPortRange(startPort int, endPort int) {
 	IptableCmd("-A", "INPUT", "-p", "tcp", "--dport", portStr, "-j", "REJECT", "--reject-with", "tcp-reset")
 	Ip6tableCmd("-A", "INPUT", "-p", "tcp", "--dport", portStr, "-j", "REJECT", "--reject-with", "tcp-reset")
 }
+
+func allowControlFlows() {
+	// Allow HTTP, ssh and guacamole packets
+	// Pick flow marking values 1, 2, 3 from the reserved space.
+	portStr := "8080,22,4822"
```

Comment:
Why do we want to allow 8080 and 4822?

Reply:
We only allow localhost 4822; never remote.

Reply:
We allow all these control packets from PREROUTING in raw table and let INPUT hook of filter table make the decision. Intent is to avoid dropping control packets destined to dom0 and not make filtering decisions.

Reply:
At a minimum the comment is completely wrong, since it says allow.

But where is the rule which matches mark=1?

Reply:
I'll re-work the comment part and function name.

Reply:
done